### ISSUE TITLE
Graphics are not shown

### ISSUE THREAD

**Issue by jasonLaster** [RecordReplay/devtools#3378]
I think this is a good example of using Replay with a slower internet and the graphics grinding to a halt. At a minimum we should show a loading state...

https://app.replay.io/recording/7543e5b5-1d41-4b2f-91aa-0527da425718?point=66526303503024973345244140628083866&time=23397.673769839515&hasFrames=true

**Comment by loganfsmyth** [comment]
I'm seeing this too: https://app.replay.io/recording/ade40853-6a11-4d67-bb2a-6c8873e6574f?point=28882151279626904067913185555907315&time=11276&hasFrames=false

**Comment by hbenl** [comment]
@loganfsmyth in your recording you have the videoPlayback feature enabled but since it is disabled in the backend, it can't show you a video.

**Comment by hbenl** [comment]
@jasonLaster if you seek to the end of your recording and evaluate `outstanding()` in the replay console, you'll see that 123 messages did not receive replies from the backend, one of which is a `Graphics.getPaintContents` request, so I think that's the issue here.

**Comment by hbenl** [comment]
While the 2 linked recordings did not show the issue that we've been seeing (that sometimes no screenshot is shown after opening a recording), I managed to reproduce it. It's very timing dependent, for me it only occurred when:
- I opened a recording URL that contains an execution point to jump to (so it didn't happen when I opened a recording from the library)
- it was a cold start (i.e. no controller for that recording was already running, so it didn't happen after a reload)
- I did _not_ have the browser's devtools open :angry:
- I was _not_ making a recording of it :rage:

In that case, devtools called `getGraphicsAtTime()` before the paint points were received from the backend.
